diff --git a/node_modules/@react-native-seoul/kakao-login/ios/RNKakaoLogins/RNKakaoLogins.swift b/node_modules/@react-native-seoul/kakao-login/ios/RNKakaoLogins/RNKakaoLogins.swift
index 202e945..d4abb06 100644
--- a/node_modules/@react-native-seoul/kakao-login/ios/RNKakaoLogins/RNKakaoLogins.swift
+++ b/node_modules/@react-native-seoul/kakao-login/ios/RNKakaoLogins/RNKakaoLogins.swift
@@ -249,25 +249,8 @@ class RNKakaoLogins: NSObject {
 
                     var result: [String: Any] = [:]
 
-                    if let useId = userServiceTerms?.id {
-                        result["userId"] = useId
-                    }
-
-                    if let serviceTerms = userServiceTerms?.serviceTerms {
-                        result["serviceTerms"] = serviceTerms.map {
-                            var terms = [
-                                "tag": $0.tag,
-                                "agreed": $0.agreed,
-                                "required": $0.required,
-                                "revocable": $0.revocable
-                            ]
-
-                            if let agreedAt = $0.agreedAt {
-                                terms["agreedAt"] = dateFormatter.string(from: agreedAt)
-                            }
-
-                            return terms
-                        }
+                    if let userId = userServiceTerms?.userId {
+                        result["userId"] = userId
                     }
 
                     resolve(result)
